src/app/features/post/pages/post-add/post-add.component.scss

```diff
@@ -0,0 +1,35 @@
+.post-add {
+	position: relative;
+	margin: 5vw auto 0;
+	width: 70vw;
+
+	border-radius: 23px;
+	background: #ffffff;
+	box-shadow: 0 1px 28px 2px rgba(97, 97, 93, 0.88);
+	padding: 32px;
+	form {
+		display: flex;
+		flex-direction: row;
```

Comment:
It will be same result without this line. `flex-direction` default value is _row_.

Reply:
done